docs/source/commandline/commandline-all.rst

```diff
@@ -1233,31 +1356,31 @@ Core Misc Options
 .. _mame-commandline-drc:
 
 **-[no]drc**
-	Enable DRC cpu core if available.  The default is ON (*-drc*).
+	Enable DRC cpu core if available. The default is ON (*-drc*).
```

Comment:
Probably worth explaining that this is a dynamic recompiler.

Reply:
Fixed on my side.